### PR TITLE
New package: autofirma-1.8.2

### PR DESCRIPTION
**Pull request by Eloitor**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-musl)

Closes https://github.com/void-linux/void-packages/issues/36872

### CODE REVIEW

File: srcpkgs/autofirma/template [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Template file for 'autofirma'
+pkgname=autofirma
+version=1.8.2
+revision=1
+_jmulticard_version=1.8
+
```
Comment (classabbyamp) [inline srcpkgs/autofirma/template:6]:
```suggestion
```

File: srcpkgs/autofirma/template [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+hostmakedepends="apache-maven openjdk11"
+depends="virtual?java-environment"
+short_desc="Cliente de firma electrónica ofrecido por la Administración Pública"
+maintainer="Eloi Torrents <eloitor@disroot.org>"
+license="GPL-3.0-or-later"
+homepage="https://firmaelectronica.gob.es/"
+distfiles="https://github.com/ctt-gob-es/clienteafirma/archive/v${version}.tar.gz
+	https://github.com/ctt-gob-es/jmulticard/archive/refs/tags/v${_jmulticard_version}.tar.gz"
```
Comment (classabbyamp) [inline srcpkgs/autofirma/template:14]: indent this with a space

File: srcpkgs/autofirma/template [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+do_check() {
+	cd "jmulticard-${_jmulticard_version}"
+	mvn test
+	cd "../clienteafirma-${version}"
+	mvn test
+}
+
+do_install() {
+	vmkdir usr/share/java
+	vmkdir usr/share/java/${pkgname}
```
Comment (classabbyamp) [inline srcpkgs/autofirma/template:38]: you don't need to vmkdir the parent and the child, it does mkdir -p internally

**Comment by Eloitor** [comment]
Fixed. I also translated the description.

**Pull request merged**